src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,392 @@
+- [Webots User Guide](https://cyberbotics.com/doc/guide/index)
+
+**Resources:**
+
+- [GitHub: NUWebots](https://github.com/NUbots/NUWebots)
+- [GitHub: Webots (Official)](https://github.com/cyberbotics/webots)
+- [GitHub: Webots (TC Version)](https://github.com/RoboCup-Humanoid-TC/webots)
+
+</details>
+
+<details>
+<summary> Mathematics </summary>
+
+Mathematics is integral part of any robotics system.
+
+**Reading:**
+
+- [NUbook: Mathematics](/system/foundations/mathematics#linear-algebra)
+- [Eigen C++ Linear Algebra Library](http://eigen.tuxfamily.org/dox/)
+
+**Videos:**
+
+- [3Blue1Brown: Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)
+
+**Tutorials:**
+
+- [MIT Open Courseware: Linear Algebra](https://ocw.mit.edu/courses/mathematics/18-06-linear-algebra-spring-2010/video-lectures/)
+
+**Resources:**
+
+- [Wolfram Alpha](http://mathworld.wolfram.com/)
+
+</details>
+
+<details>
+<summary> Python </summary>
+
+Python is used in the build system in the main robot codebase.
+
+**Videos:**
+
+- [LinkedIn Learning: Python Essential Training](https://www.linkedin.com/learning/python-essential-training-2/)
+
+</details>
+
+<details>
+<summary> Machine Learning </summary>
+
+Primarily used in the vision system, but could be used in any system.
+
+**Videos:**
+
+- [YouTube: 3Blue1Brown: Neural Networks](https://www.youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi)
+- [YouTube: DeepLizard: Machine Learning & Deep Learning Fundamentals](https://www.youtube.com/watch?v=gZmobeGL0Yg&list=PLZbbT5o_s2xq7LwI2y8_QtvuXZedL6tQU)
+
+</details>
+
+<details>
+<summary> Log and Play Back Data </summary>
+
+**Reading:**
+
+- [NUbook: Data Recording and Playback Guide](/guides/main/data-recording-playback)
+- [NUbook: Logging](/system/subsystems/logging)
+
+</details>
+
+<details>
+<summary> Lab Induction </summary>
+
+The lab induction must be completed before getting swipe card access to the lab.
+
+**Reading:**
+
+- [NUbook: Lab Induction](/team/induction)
+
+**Tutorials:**
+
+- NUbots lab inductions are run for new team members after they join.
+
+</details>
+
+## Behaviour
+
+<details>
+<summary>General</summary>
+
+**Reading:**
+
+- [NUbook](/system/subsystems/behaviour)
+
+**Resources:**
+
+- [GitHub: GameController](https://github.com/RoboCup-Humanoid-TC/GameController)
+- [GitHub: Communication Protocol](https://github.com/RoboCup-Humanoid-TC/RobocupProtocol)
+
+</details>
+
+## DevOps and Quality Assurance
+
+<details>
+<summary> The Build System </summary>
+
+**Reading:**
+
+- [NUbook: Build System](https://nubook.nubots.net/system/foundations/build-system)
+- [NUbook: Configuration and Script System](/system/foundations/config-script)
+- [Modern CMake](https://cliutils.gitlab.io/modern-cmake/)
+- [ArchWiki](https://wiki.archlinux.org/)
+- [Book: Docker Deep Dive](https://www.amazon.com.au/Docker-Deep-Dive-Nigel-Poulton-ebook/dp/B01LXWQUFF)
```

Comment:
Is amazon the way we want to link books?

Reply:
Down with amazon >:(

Reply:
Probably not, it's just the link I got from Yoshi

Reply:
I can't find any great link. Options I can see:
- Google (only link I can find that isn't a seller): https://books.google.com.au/books/about/Docker_Deep_Dive.html?id=860yswEACAAJ
- Book Repository (cheap hardcopy): https://www.bookdepository.com/Docker-Craig-Berg/9798657599404
- Booktopia (cheap ebook): https://www.booktopia.com.au/docker-deep-dive-nigel-poulton/ebook/1230004026020.html

Reply:
I think google books link is good